src/main/java/Objects/NetworkDevices/NetworkDevice.java

```diff
@@ -19,13 +28,49 @@ public String getId() {
         return id;
     }
 
+    public void setPacket(final String packet_type) {
+        packets.add(packet_type);
+    }
+
+    public List getPacket() {
+        return packets;
+    }
+
+    public Boolean isHidden() {
+        return hidden;
+    }
+
+    public void setHidden(final Boolean hidden) {
+        this.hidden = hidden;
+    }
+
+    public void setMaxPacket(final int max_packet) {
+        this.max_packet = max_packet;
+    }
+
+    public int getMaxPacket() {
+        return max_packet;
+    }
+
     public void setTeam(final String team) {
         this.team = team;
     }
 
     public String getTeam() {
         return team;
     }
+
+    public void setSpeed(final int speed) {
+        this.speed = speed;
+    }
+
+    public void setConnections(final ArrayList<Integer> connections) {
+        this.connections = connections;
+    }
+
+    public ArrayList<Integer> getConnections() {
+        return connections;
+    }
     //TODO: other universal values?
 
```

Comment:
my god

Reply:
Calm down, Martha Stewart.  Once it's compiled, it won't matter.  This isn't Python.